x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -1,55 +1,22 @@
 [[configuring-stack-security]]
 == Start the Elastic Stack with security enabled
```

Comment:
I Think we had this discussion in a previous PR too. I find the extensive introduction which repeats a large part of the information that we  include below in [[stack-start-with-security]] too, a bit confusing, Maybe it's just me thought, if we think this is helpful for users, +1 to leave as is. Just wanted to raise  this and we can iterate for GA if need be